iree/hal/vulkan/vulkan_device.h

```diff
@@ -31,19 +31,30 @@
 #include "iree/hal/vulkan/extensibility_util.h"
 #include "iree/hal/vulkan/handle_util.h"
 #include "iree/hal/vulkan/legacy_fence.h"
+#include "iree/hal/vulkan/queues_info.h"
 
 namespace iree {
 namespace hal {
 namespace vulkan {
 
 class VulkanDevice final : public Device {
  public:
+  // Creates a device that manages its own VkDevice.
   static StatusOr<ref_ptr<VulkanDevice>> Create(
       ref_ptr<Driver> driver, const DeviceInfo& device_info,
       VkPhysicalDevice physical_device,
       const ExtensibilitySpec& extensibility_spec,
       const ref_ptr<DynamicSymbols>& syms);
 
+  // Creates a device that shares an externally managed VkDevice.
+  static StatusOr<ref_ptr<VulkanDevice>> Create(
```

Comment:
rename `Wrap` to indicate that we are wrapping an externally owned device (not `Create`ing one)